models/engine/file_storage.py

```diff
@@ -12,59 +10,74 @@
+		if cls is not None:
+			new_dict = {}
+			for key, value in self.__objects.items():
+				if cls == value.__class__ or cls == value.__class__.__name__:
+					new_dict[key] = value
+			return new_dict
+		return self.__objects
 
-    def all(self, cls=None):
-        """returns the dictionary __objects"""
-        if cls is not None:
-            new_dict = {}
-            for key, value in self.__objects.items():
-                if cls == value.__class__ or cls == value.__class__.__name__:
-                    new_dict[key] = value
-            return new_dict
-        return self.__objects
+	def new(self, obj):
+		"""sets in __objects the obj with key <obj class name>.id"""
+		if obj is not None:
+			key = obj.__class__.__name__ + "." + obj.id
+			self.__objects[key] = obj
 
-    def new(self, obj):
-        """sets in __objects the obj with key <obj class name>.id"""
-        if obj is not None:
-            key = obj.__class__.__name__ + "." + obj.id
-            self.__objects[key] = obj
+	def save(self):
+		"""serializes __objects to the JSON file (path: __file_path)"""
+		json_objects = {}
+		for key in self.__objects:
+			json_objects[key] = self.__objects[key].to_dict()
+		with open(self.__file_path, 'w') as f:
+			json.dump(json_objects, f)
 
-    def save(self):
-        """serializes __objects to the JSON file (path: __file_path)"""
-        json_objects = {}
-        for key in self.__objects:
-            json_objects[key] = self.__objects[key].to_dict()
-        with open(self.__file_path, 'w') as f:
-            json.dump(json_objects, f)
+	def reload(self):
+		"""deserializes the JSON file to __objects"""
+		try:
+			with open(self.__file_path, 'r') as f:
+				jo = json.load(f)
+			for key in jo:
+				self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
+		except:
+			pass
 
-    def reload(self):
-        """deserializes the JSON file to __objects"""
-        try:
-            with open(self.__file_path, 'r') as f:
-                jo = json.load(f)
-            for key in jo:
-                self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except:
-            pass
+	def delete(self, obj=None):
+		"""delete obj from __objects if it’s inside"""
+		if obj is not None:
+			Key = obj.__class__.__name__ + '.' + obj.id
+			if key in self.__objects:
+				del self.__objects[key]
 
-    def delete(self, obj=None):
-        """delete obj from __objects if it’s inside"""
-        if obj is not None:
-            key = obj.__class__.__name__ + '.' + obj.id
-            if key in self.__objects:
-                del self.__objects[key]
+	def close(self):
+		"""call reload() method for deserializing the JSON file to objects"""
+		self.reload()
+	
+	def count(self, cls=None):
+		""""return the obect base on the class name and its ID else None"""
+		#for k,v in self.all(cls).items():
+		#print(v)
+		if cls in classes.values():
+			return len(self.all(cls))
+		elif cls is None:
+			return len(self.all())
```

Comment:
Very straight forward and understandable method. I would recommend using comments although it is not necessary in this case since it is understandable.